ServerlessLibraryAPI/ClientApp/src/components/Contribute/ContributionsList.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Label } from "office-ui-fabric-react";
+import ItemList from "../../components/ItemList/ItemList";
+
+class ContributionsList extends Component {
+  filteredSamples() {
+    let { samples, user } = this.props;
+    let { userName } = user || "__nouser__";
+    let filter = new RegExp(userName, "i");
+    samples = samples.filter(
+      el => el.repository.replace("https://github.com/", "").match(filter) // this match should be against author
+    );
+
+    samples = samples.sort(function(a, b) {
+      var titleA = a.title.toLowerCase(),
+        titleB = b.title.toLowerCase();
+      if (titleA < titleB)
+        //sort string ascending
+        return -1;
+      if (titleA > titleB) return 1;
+      return 0; //default return value (no sorting)
+    });
+
+    return samples;
+  }
+
+  render() {
+    const headerLabelStyles = {
+      root: {
+        fontSize: "12px",
+        fontWeight: "bold",
+        paddingTop: "0px",
```

Comment:
is it required?

Reply:
This control has a non-zero default padding, which I want to remove